### DISCUSSION TITLE
How to disable 2 factor authentication? I haven't enabled it.

### DISCUSSION THREAD

**Discussion by cpypcy** [community/community#37205] [Other Feature Feedback, Questions, & Ideas]
How do I disable 2 factor authentication when I never enabled it in first place? In settings it says it's disabled but I still have to login with some email code that is being sent to me. I never enabled this crap in first place! Let me just login like a normal person.

**Comment by DSDmark** [comment]
Hey @cpypcy,

if you want to **disable** that 2-factor authentication, then you can do like this:-

Step 1: open your settings.

- ![Screenshot from 2022-10-25 17-07-15](https://user-images.githubusercontent.com/85744816/197764260-d3c9f9b3-94f4-44c0-974a-51bcf501c705.png)

Step 2: inside that settings you can see the **access** section on that section there is a **password and authentication** click on that, and you can see the **button** to **Enabled** or **disabled** that feature.

- ![Screenshot from 2022-10-25 17-08-04](https://user-images.githubusercontent.com/85744816/197764421-5c595737-df4c-4ad3-b79b-2187731699ea.png)

> it's your choice to do that but i not remmond you for that 

And also if it's show you **disabled,** then add some details like screenshot.
